modules/fx/arrow-fx/src/main/kotlin/arrow/fx/typeclasses/Bracket.kt

```diff
@@ -22,6 +22,19 @@ sealed class ExitCase<out E> {
 fun <E> Either<E, *>.toExitCase() =
   fold(::Error) { ExitCase.Completed }
 
+sealed class ExitCase2<out E> {
```

Comment:
Not a fan of versioning in the name, if could be `ExitCases` or something similar.

Reply:
Not a fan of the name either 😅

I was hoping to do something with `sealed class ExitCase2<out E, out E2> : ExitCase<E>()` but that's weird since `ExitCase` has `Error` which should be `E2` according to the current `IO` terminology 🤕